lang/rust/avro/examples/test_interop_single_object_encoding.rs

```diff
@@ -58,3 +63,15 @@ fn main() {
         .expect("Should encode");
     assert_eq!(file_message, generated_encoding)
 }
+
+fn test_read() {
+    let file_message = std::fs::read(format!("{}/test_message.bin", RESOURCES_FOLDER))
```

Comment:
Is this `read` really needed ?

Reply:
I can reuse. I'll just need to clone because `test_read` needs the owned value

Reply:
just kidding, I can use a reference for test write

Reply:
I didn't understand the intention last evening! All is fine!